core/src/test/scala/org/apache/spark/scheduler/SchedulerIntegrationSuite.scala

```diff
@@ -27,6 +27,8 @@ import scala.language.existentials
 import scala.reflect.ClassTag
 
 import org.scalactic.TripleEquals
+import org.scalatest.concurrent.Eventually._
+import org.scalatest.time.SpanSugar._
 import org.scalatest.Assertions.AssertionsHelper
```

Comment:
no problem, sorry I never get these right -- I actually thought package imports were supposed to be first and checked some other places to realize you are right.